hr_timesheet_activity_edit_top/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1 @@
+Add new activity from the top of the list. This is useful when quickly adding new timesheet entries.
```

Comment:
```suggestion
Add new timesheet entries on the top of the list instead of on the bottom. This is useful when quickly adding new timesheet entries.
```